### PR TITLE
Matt/update beats port input

### PR DESCRIPTION
**Pull request by mattedgedelta**
Added filebeats input information.

### CODE REVIEW

**Review by yildizfatih — APPROVED** [review]

File: docs/configuration/inputs.md [hunk 1]
```diff
@@ -314,4 +315,15 @@ Agent component health data contains information about agent's internal state an
 ```yaml
   agent_components_health:
     labels: "agent-components-health"
+```
+
+## Elastic Beats
```
Comment (taylanisikdemir) [inline docs/configuration/inputs.md:320]: a side note: we don't usually add scenarios like this to inputs.md and keep one section per input type (type mapping 1-1 to yml types) on this page. In reality there can be many use cases that syslog tcp port can be used for and they should probably be listed somewhere else (currently we add stuff like this to appendices but that's not ideal) or we repurpose inputs.md to be more use-case oriented. 
anyway we are going to re-organize whole documentation after George (new tech writer) joins next month.

File: docs/configuration/inputs.md [hunk 2]
```diff
@@ -314,4 +315,15 @@ Agent component health data contains information about agent's internal state an
...
+
+If enabled, Elastic Beats allows you to specify an endpoint to have the agent listen on for incoming traffic from filebeats. This functionality is used in conjunction with the Logstash output configuration of filebeats.
```
Comment (taylanisikdemir) [inline docs/configuration/inputs.md:322]: an example logstash output config would be nice to show here. I believe it's supposed to have new line codec. correct me if I'm wrong @orhanasan

**Review by taylanisikdemir — APPROVED** [review]

File: docs/configuration/inputs.md (hunk 1 above)
Comment (mattedgedelta) [reply docs/configuration/inputs.md:320]: I completely understand. This is for a specific ask from RBC to include in documentation.

File: docs/configuration/inputs.md (hunk 2 above)
Comment (mattedgedelta) [reply docs/configuration/inputs.md:322]: Added filebeats.yml config for logstash output.

**Pull request merged**